ember.js/2016-08/august-19.md

```diff
@@ -0,0 +1,248 @@
+
+YK: DHH's 15 minutes video is not compatible with the way we share and talk about things to today. I agree.
+
+TD: They are also linkable, each section would be "I want to think about this"
+
+YK: If you want to make smaller videos you need to think about what discrete chunk of knowledge you're teaching
+
+RJ: Unless I missed it, also this does not talk about tests
+
+YK: It is hard to do tests in 15 minutes
+
+TD: If you chunk it up, you do it as build the feature and test it.
+
+YK: I think if the chunks get too long (10m) it seems difficult
+
+GC: Five minutes sounds better
+
+TD: The marketing message angular had with testing was really powerful early on. We can probably use that to our advantage now especially if we have the improved testing items.
+
+GC: Anything we can do to keep it up to date?
+
+YK: For me screenflow has been the only thing useful. I'd like a tool that let you script this. Applescript can do stuff like this.
+
+YK: If you restrict it you could likely build something. Making it generic is what is the pain in an ass.
+
+#### RFC for transition event
+
+RJ: I'd like to FCP this
+
+YK: I have some blocking thoughts.
+
+(discussion)
+
+YK: There are several hooks in that space, and I'm hesitant to add another one without understanding the impact on teaching, or knowing how to steer people away from it when they shouldn't land there.
+
+KS: It overlaps with the routing service right?
+
+EF: There are things in this RFC that take private objects and makes them effectively public. They would also need to be documented and thought about.
+
+YK: Ed does your RFC cover this case?
+
+EF: Not as written, no
+
+(discussion)
+
+KS: The router itself being an interface is kind of hap-hazard. This RFC exists b/c a side effect is important for tracking but it isn't observable. In some of my apps we're overriding a private method to get access to things.
+
+YK: I think the routing service is the better place to add cross-cutting items
+
+MB: I will make some time to help Edward push it forward.
+
+RJ: There is a decent amount of work implementing a chunk of the RFC as it is currently written. The bigger issue is getting the design stuff finished. The stuff implemented doesn't need to be re-done.
+
+EF: One of my takeaways from last time was that the link-tos should not be using the service for sideways bindings, it should be coupled to the outlet states.
+
+KS: By doing the outlet state we also have the timing we expect visually with the URL.
+
+EF: Right is helps timings for animations and improves performance.
+
+YK: Seems like main action is to move the routing service RFC forward. The author should contribute to the routing RFC.
+
+EF: Copying the current API leaks into the public/intimate API surface shouldn't be done. I think this is "scratch and itch" and that is why the RFC might be getting a bad reaction.
+
+EF: If he wants it to be separate as an RFC that is ok, but it should be in terms of the routing RFC.
```

Comment:
It would be great to see feedback like this on the RFC itself.

As far as the transition event RFC being written in terms of the routing RFC, can someone please elaborate?

emberjs/rfcs#95 (the router RFC) states that [`Router#willTransition` "will be carried over verbatim to `RouterService`"](https://github.com/emberjs/rfcs/blame/router-service/text/0000-router-service.md#L56-L64). The [text](https://github.com/nickiaconis/rfcs/blame/redirect-notify/text/0000-transition-hooks-events.md#L72) and [examples](https://github.com/nickiaconis/rfcs/blame/redirect-notify/text/0000-transition-hooks-events.md#L75-L81) of emberjs/rfcs#126 (the transition event RFC) are written in terms of the [`willTransition` API](https://github.com/emberjs/ember.js/blob/v2.7.0/packages/ember-routing/lib/system/router.js#L292-L308).

Assuming emberjs/rfcs#95 truly intends to carry over the `willTransition` API verbatim, it stands to reason that emberjs/rfcs#126 is written in terms of both emberjs/rfcs#95 and the existing Router API.

Some mental find/replace of Ember.Router -> Ember.RouterService may be required, and I can update those references if that's really what's blocking it, but it seems a rather trivial reason to prevent emberjs/rfcs#126 from moving forward, so I hope I'm simply missing some key piece of information.